main.cpp

```diff
@@ -108,8 +108,55 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+    struct Foot
+    {
+        int footYPosition = 0;
+        int size = 3;
 
+        int stepSize();
+        void stepForward();
+    };
 
+    void run( int howFast, bool startWithLeftFoot);
+
+    Foot leftFoot;
+    Foot rightFoot;
+};
+
+int Person::Foot::stepSize()
+{
+    return size;
+}
+
+void Person::Foot::stepForward()
+{
+    footYPosition = 3;
+}
+
+void Person::run( int howFast, bool startWithLeftFoot)
+{
+    if( startWithLeftFoot == true )
+    {
+        leftFoot.stepForward();
+        rightFoot.stepForward();
+    }
+    else
+        {
+        rightFoot.stepForward(); // returns void
```

Comment:
Fix this curly brace indentation
- [ ] revise

Reply:
this was aligned already on my end oddly, IDK.